gapis/api/vulkan/api/descriptor.api

```diff
@@ -661,26 +661,23 @@ cmd void vkUpdateDescriptorSets(
           VK_DESCRIPTOR_TYPE_SAMPLED_IMAGE,
           VK_DESCRIPTOR_TYPE_STORAGE_IMAGE,
           VK_DESCRIPTOR_TYPE_INPUT_ATTACHMENT: {
-        imageBinding := dstBinding.ImageBinding
-        imageBinding[c.DstArrayIndex] =
-        srcBinding.ImageBinding[c.SrcArrayIndex]
-        dstBinding.ImageBinding = imageBinding
 
         if srcBinding.ImageBinding[c.SrcArrayIndex] == null {
-          vkErrorInvalidSampler(null)
+           vkErrInvalidDescriptorArrayElement(as!u64(c.SrcSet), c.SrcBinding, c.SrcArrayIndex)
         } else {
+          imageBinding := dstBinding.ImageBinding
+          imageBinding[c.DstArrayIndex] =
+          srcBinding.ImageBinding[c.SrcArrayIndex]
```

Comment:
can you combine these two lines? i know they were separate before, but they really shouldn't be.... or at the very least indent the second one.

Reply:
Surely.